python/tank/descriptor/bundle_cache_usage/exception.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2017 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+
+class BundleCacheUsageException(Exception):
```

Comment:
See rest of toolkit code and follow that standard. Derive from `TankDescriptorError` - see https://github.com/shotgunsoftware/tk-core/blob/master/python/tank/descriptor/errors.py#L19. Suffix with Error, not exception.